File: LAB QUIZZES/CSE221_Quiz03/_A3A/_A3A_Testing.py
import os, sys, ast, time, random, threading, subprocess
# Compile and run: pypy _A3A_Testing.py
N = 0; res = 0
A = []
def solve(A):
    N = len(A)
    res = 0
    # Place your code here
    return res

# # file input
# # with open("TimeLimitExceeded.txt") as fin:
# # with open("CustomTest.txt") as fin:
# with open("WrongAnswer.txt") as fin:
#     for i in range(3):
#         fin.readline()
#     A = ast.literal_eval(fin.readline()[4:])

# stdin
# 5
# 1 2 5 4 3
N = int(input())
A = list(map(int, input().split()))

# # hardcoded input
# N = 5
# A = [1, 2, 5, 4, 3]

start = time.time()
res = solve(A.copy())
finish = time.time()
elapsed = finish - start

# # file output
# with open("OutputOfYourCode.txt", "w") as fout:
#     fout.write(str(res))

# stdout
print(res)

print("Execution time for the targeted one test is " + f"{elapsed:.9f}" + "s", flush=True)
os._exit(0)

File: LAB QUIZZES/CSE221_Quiz03/_A3A/_A3A_Testing.java
import java.util.*;
import java.io.*;

// Compile and run: javac _A3A_Testing.java && java _A3A_Testing
public class _A3A_Testing {
    public static int N = 0;
    public static ArrayList<Integer> A = new ArrayList<>();
    public static Long res = 0L;

    public static Long solve(ArrayList<Integer> A) {
        int N = A.size();
        Long res = 0L;
        // Place your code here
        return res;
    }

    public static void main(String args[]) throws Exception {
        // // file input
        // // BufferedReader fin = new BufferedReader(new
        // // FileReader("TimeLimitExceeded.txt"));
        // // BufferedReader fin = new BufferedReader(new FileReader("CustomTest.txt"));
        // BufferedReader fin = new BufferedReader(new FileReader("WrongAnswer.txt"));
        // for (int i = 1; i <= 3; ++i)
        // fin.readLine();
        // fin.skip(5);
        // for (String temp : fin.readLine().replace("]", "").split(", "))
        // A.add(Integer.parseInt(temp));
        // fin.close();

        // stdin
        // 5
        // 1 2 5 4 3
        BufferedReader fin = new BufferedReader(new InputStreamReader(System.in));
        String[] tempA = fin.readLine().split(" ");
        N = Integer.parseInt(tempA[0]);
        for (String temp : fin.readLine().split(" "))
            A.add(Integer.parseInt(temp));
        fin.close();

        // // hardcoded input
        // N = 5;
        // A = new ArrayList<>(Arrays.asList(1, 2, 5, 4, 3));

        long start = System.nanoTime();
        res = solve(new ArrayList<>(A));
        long finish = System.nanoTime();
        long elapsed = finish - start;

        // // file output
        // BufferedWriter fout = new BufferedWriter(new
        // FileWriter("OutputOfYourCode.txt"));
        // fout.write(res.toString());
        // fout.close();

        // stdout
        System.out.println(res);

        System.out.printf("Execution time for the targeted one test is %.9fs\n", elapsed * 1e-9);
        System.exit(0);
    }
}
